Assets/MixedRealityToolkit/_Core/Definitions/MixedRealityConfigurationProfile.cs

```diff
@@ -103,6 +104,29 @@ public MixedRealitySpeechCommandsProfile SpeechCommandsProfile
             private set { speechCommandsProfile = value; }
         }
 
+        [SerializeField]
+        [Tooltip("Enable and configure the devices for your application.")]
+        private bool enableControllerProfiles = false;
+
+        /// <summary>
+        /// Enable and configure the devices for your application.
+        /// </summary>
+        public bool EnableControllerProfiles
+        {
+            get { return enableControllerProfiles; }
+            private set { enableControllerProfiles = value; }
+        }
+
+        [SerializeField]
+        [Tooltip("Device profile for wiring up physical inputs to Actions.")]
+        private MixedRealityControllerMappingProfile controllersProfile;
+        public MixedRealityControllerMappingProfile ControllersProfile
```

Comment:
missing summary

Reply:
Grr, that was your property, but my bad for not documenting it when I grabbed it